server/routes/message.js

```diff
@@ -0,0 +1,46 @@
+var express = require("express");
+var router = express.Router();
+
+var {mongoose, db, Message, User} = require('../database');
+
+router.get("/:id", function(req, res, next) {
+    Message.findById(req.params.id, function(err, message){
+        res.json(message);
+    });
+});
+
+//Send a message
+router.post("/", function(req, res, next) {
```

Comment:
It doesn't seem like this actually works yet, none of my messages show up in the database. I think it could be related to the first issue where it's refreshing before it can actually send a message